_providers/sdf.org.md

```diff
@@ -0,0 +1,24 @@
+---
+name: sdf.org
+status: PREPARATION
+domains:
+  - sdf.org
+server:
+  - type: imap
+    socket: SSL
+    hostname: mx.sdf.org
+    port: 993
+    username: EMAILLOCALPART
+  - type: smtp
+    socket: STARTTLS
+    hostname: smtp.example.com
+    port: 587
+    username: EMAILADDRESS
```

Comment:
according to https://sdf.org/?tutorials/smtpauth , that should be a so-called _domain_, iiuc for username@sdf.org that might be username.sdf.org@sdf.org - or sth. else. so, the email address seems to be wrong here.

i would just mention that special bit in the before-login-hint, saying eg.

```
In addition to email and password, you have to fill out 'Advanced / SMTP login-name'.
Typicaly, this is USERNAME.sdf.org@sdf.org if your e-mail-address is USERNAME@sdf.org.
All other fields can be empty.
```